crates/turborepo-lib/src/shim/local_turbo_config.rs

```diff
@@ -7,6 +7,7 @@ use jsonc_parser::{
 use tracing::debug;
 use turborepo_repository::{inference::RepoState, package_manager::PackageManager};
 
+const TURBO_DOWNLOAD_LOCAL_ENABLED: &str = "TURBO_DOWNLOAD_LOCAL_ENABLED";
 const TURBO_DOWNLOAD_LOCAL_DISABLED: &str = "TURBO_DOWNLOAD_LOCAL_DISABLED";
```

Comment:
Do we need both? If/when we make it opt out, can we just require people to set `_ENABLED=false`?